kodilla-stream/src/test/java/com/kodilla/stream/array/ArrayOperationsTestSuite.java

```diff
@@ -0,0 +1,53 @@
+package com.kodilla.stream.array;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.Random;
+
+public class ArrayOperationsTestSuite {
+
+
+    public int[] generateArray(int n) {
+        Random rn = new Random();
```

Comment:
generally, using random values in test is not the best option. Let assume that you generate such input data, that your test will fail. How will you find you for which data your test fails? How do you repeat test after fixing problem? Test has to be REPRODUCIBLE.